app/controllers/users_controller.rb

```diff
@@ -3,6 +3,27 @@ class UsersController < ApplicationController
   before_action :retrieve_patients, only: [:add_patient, :remove_patient]
   rescue_from Mongoid::Errors::DocumentNotFound, with: -> { head :bad_request }
 
+  def create
+    raise 'Permission Denied' unless current_user.admin?
```

Comment:
Since `UsersController#new` already deflects an unauthorized user I think raising an error is a better response for `UsersController#create` since something must be REALLY WRONG if a non-admin user is able to get through to the create route.